frontend/packages/ux-editor/src/components/config/expressions/ExpressionContent.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { Button, ButtonColor, ButtonVariant, Select, ToggleButtonGroup } from '@digdir/design-system-react';
+import { ExpressionFunction } from '../../../types/Expressions';
+import { XMarkIcon } from '@navikt/aksel-icons';
+import cn from 'classnames';
+import classes from './ExpressionContent.module.css';
+
+export interface IExpressionContentProps {
+  expressionAction: boolean;
+  expressionElement: ExpressionElement;
+  onAddExpressionElement: () => void;
+  onUpdateExpressionElement: (expressionElement: ExpressionElement) => void;
+  onRemoveExpressionElement: (expressionElement: ExpressionElement) => void;
+}
+
+export interface ExpressionElement {
+  id: string;
+  expressionOperatorForPrevExpression?: 'Og' | 'Eller'; // Or is it for the next expression?
```

Comment:
The way you have implemented it here (just looked at the console.log of the dynamics object) it seems to be for the next expression 🤔